pkg/R/sparkR.R

```diff
@@ -15,7 +15,7 @@ sparkR.onLoad <- function(libname, pkgname) {
 # backend and FALSE otherwise
 connExists <- function(env) {
   tryCatch({
-    exists(".sparkRCon", envir = env) && isOpen(env[[".sparkRCon"]])
+    !exists(".sparkRCon", envir = env) || (exists(".sparkRCon", envir = env) && isOpen(env[[".sparkRCon"]]))
```

Comment:
@sun-rui I think oscaroboto is on to the something, but the additional logic verges on tautology.  I think changing the connExists function is a more straightforward test for the same problem.